apolo-sdk/src/apolo_sdk/_server_cfg.py

```diff
@@ -179,6 +186,14 @@ def _parse_cluster_config(payload: Dict[str, Any]) -> Cluster:
         orgs = ["NO_ORG"]
     else:
         orgs = [org if org is not None else "NO_ORG" for org in orgs]
+
+    apps_payload = payload.get("apps", {})
+    apps_config = AppsConfig()
+    if apps_payload:
+        apps_config = AppsConfig(
+            apps_hostname_templates=apps_payload.get("apps_hostname_templates", [])
+        )
```

Comment:
Let's avoid calling `AppsConfig` constructor twice.
```suggestion
    if apps_payload:
        apps_config = AppsConfig(
            apps_hostname_templates=apps_payload.get("apps_hostname_templates", [])
        )
    else:
        apps_config = AppsConfig()
```

Reply:
applied, thank you!